src/Pos/Update/Poll/Trans.hs

```diff
@@ -1,3 +1,8 @@
+{-
```

Comment:
Ah, you're right. I commented it out after a merge request to see if the extensions were needed. Guess they weren't. I'm currently fixing stuff after Ivan's review, I'll fix this next.